### PR TITLE
Rubocop. Use default configuration.

### PR DESCRIPTION
**Pull request by patrickquigley102**
By using the default configuration for Rubocop and it's extentions we're going to comply with the Ruby communities style for Ruby. Good times!

When running Rubocop now locally I get the following.

```
34 files inspected, 203 offenses detected, 171 offenses autocorrectable
```

Time to fix those!

### CODE REVIEW

**Review by awalsh3 — DISMISSED** [review]
This is ready for review!

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# # This configuration was generated by
```
Comment (patrickquigley102) [inline .rubocop_todo.yml]: Why have you included this file?

File: Gemfile [hunk 2]
```diff
@@ -49,7 +51,8 @@ end
 
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
-  gem "capybara"
-  gem "selenium-webdriver"
-  gem "webdrivers"
+  gem 'capybara'
+  gem 'factory_bot_rails'
```
Comment (patrickquigley102) [inline Gemfile:55]: It's fine for this PR but, this is scope creep ideally you should avoid.

This PR is for Rubocop compliance, which has nothing to do with Factory Bot and the test suite. They're separate concerns in my opinion.

Keeping PRs as scoped to one concern is always worth doing.

**Review by patrickquigley102 — COMMENTED** [review]
Two minor points, one with likely changes to action.

What do you think about the general approach to linters we are going for here?

File: .rubocop_todo.yml (hunk 1 above)
Comment (awalsh3) [reply .rubocop_todo.yml]: It was an option to generate a to-do list from the terminal right in the file. I did this before I saw the option to write in the to-do comments in the exact line location. I will remove!
